### DISCUSSION TITLE
sometimes, it takes more than 20 seconds by action

### DISCUSSION THREAD

**Discussion by efrenette11** [appium/appium#16653] [Q&A]
<img width="3046" alt="2022-03-24_14-36-27" src="https://user-images.githubusercontent.com/30326999/159987285-27a2b61a-f45b-496e-8fef-cd9b8ded780c.png">
As you can see in the attached image, for the exact same script, I get diffetent results. 

The one that takes 20 seconds instead of one causing my a lot of performance issues on my automated tests. 

How can I go deeper in the investogation ? Any suggestion ?

<img width="1711" alt="2022-03-24_14-55-34" src="https://user-images.githubusercontent.com/30326999/159990190-23b243cc-dce3-4d8f-a3dd-5c1986734ef2.png">

The locators is on the page 
<img width="1576" alt="2022-03-24_14-56-29" src="https://user-images.githubusercontent.com/30326999/159990426-874abc33-2846-4a10-b1db-ea898e002c6d.png">

**Comment by jlipps** [comment]
The next thing to look at would be the iOS system logs
